Dockerfile

```diff
@@ -167,8 +167,7 @@ FROM api-runtime-private as saas-api
 RUN --mount=type=secret,id=sse_pgp_pkey \
   apt-get update && apt-get install -y gnupg && \
   gpg --import /run/secrets/sse_pgp_pkey && \
-  mv /root/.gnupg /app/; \
-  chown -R nobody /app/.gnupg
+  mv /root/.gnupg/ /app/
```

Comment:
I'm keen to get @gagantrivedi's review here as well to make sure we know what we need to test.

Reply:
Sure. The image has been deployed to staging so we just need to verify SSE there.

Reply:
Yeah, we have to make sure `sse` events are still reaching influx